.htaccess

```diff
@@ -6,6 +6,7 @@
 
 <IfModule mod_rewrite.c>
     RewriteEngine on
+    RewriteRule    ^(\.well-known/.*)$ $1 [L]
```

Comment:
I would prefer this to be commented out by default. Those who need it, can uncomment it.
This file is part of the app and thus becomes userland and thus should be part of a first party version control system.

Reply:
If you're using an .htaccess file at all you have already made a performance mistake :smile: